### PR TITLE
Examples for relative/separate schema in Reference Object

### PR DESCRIPTION
**Pull request by jasonh-n-austin**
@webron Created a separate PR for updated spec examples for separate files, to polish off #384 
This was all in support of https://github.com/swagger-api/swagger-js/issues/417, which is not yet merged to master, AFAIK. Not sure if you want this in master docs until tooling supports it, but your call.

### CODE REVIEW

File: versions/2.0.md [hunk 1]
```diff
@@ -1126,6 +1126,27 @@ Field Name | Type | Description
...
+##### Relative Schema File Example
+```js
+{
+  "$ref": "Pet.json"
+}
+```
+
+```yaml
+$ref: 'Pet.yaml'
+
```
Comment (webron) [inline versions/2.0.md]: Missed closing sample (```).

File: versions/2.0.md [hunk 2]
```diff
@@ -1126,6 +1126,27 @@ Field Name | Type | Description
...
+##### Relative Files With Embedded Schema Example
+```js
+{
+  "$ref": "definitions.json#/Pet"
+}
+```
+
+```yaml
+$ref: 'definitions.yaml#/Pet'
+
```
Comment (webron) [inline versions/2.0.md]: Missed closing sample (```).

**Comment by jasonh-n-austin** [comment]
@webron that's what I get for editing Markdown without looking at it in an editor :)
Just pushed fix.

**Comment by webron** [comment]
Okay, I've merged it, but I think I'd want to add some more text there. Would you mind opening an issue to expand the explanation on JSON Reference in the spec and I'll get around to it?

**Comment by jasonh-n-austin** [comment]
Sure thing
https://github.com/swagger-api/swagger-spec/issues/388

**Comment by webron** [comment]
:+1:

**Pull request merged**